dashboard/db/migrate/20160616182304_add_school_district_other_to_enrollment.rb

```diff
@@ -0,0 +1,7 @@
+class AddSchoolDistrictOtherToEnrollment < ActiveRecord::Migration
+  def change
+    # school_district_other is true if the user explicitly stated that their
```

Comment:
Does it make sense to set this boolean to be `true` for some of the existing records?